build.sbt

```diff
@@ -180,6 +180,7 @@ lazy val `kyo-data` =
         .in(file("kyo-data"))
         .settings(
             `kyo-settings`,
+            libraryDependencies += "com.lihaoyi"   %%% "pprint"        % "0.9.0",
```

Comment:
Used by `Frame`

Reply:
Should we consider replicating the bare minimum functionality to avoid this dependency? I think we only need JCTools as a hard dependency.